cmd/admin-revoker/main_test.go

```diff
@@ -528,11 +440,14 @@ func setup(t *testing.T) testCtx {
 	if err != nil {
 		t.Fatalf("Failed to create dbMap: %s", err)
 	}
+	incidentsDbMap, err := sa.NewDbMap(vars.DBConnIncidents, sa.DbSettings{})
+	test.AssertNotError(t, err, "Couldn't create test dbMap")
+	defer test.ResetIncidentsTestDatabase(t)
```

Comment:
This should be done in testCtx.cleanup(), rather than deferred here.

Reply:
Oh gosh, idk how I did this. Good catch.